joss_paper/paper.md

```diff
@@ -0,0 +1,187 @@
+   index: 5
+ - name: UNC Eshelman School of Pharmacy, University of North Carolina, Chapel Hill, NC, USA.
+   index: 6
+ - name: Molecular Biology and Nanotechnology Laboratory (MolBNL@UniTS), DEA, University of Trieste, Trieste, Italy
+   index: 7
+ - name: Stuttgart Center for Simulation Science (SC SimTech) & Institute for Computational Physics, University of Stuttgart, 70569 Stuttgart, Germany 
+   index: 8
+ - name: University of California Irvine, Irvine, California, USA
+   index: 9
+ - name: University of Colorado Boulder, Boulder, Colorado, USA
+   index: 10
+
+date: 31 May 2024
+bibliography: paper.bib
+
+---
+
+# Summary
+
+*alchemlyb* is an open-source Python software package for the analysis of alchemical free energy calculations, an important method in computational chemistry and biology, most notably in the field of drug discovery.
+Its functionality contains individual composable building blocks for all aspects of a full typical free energy analysis workflow, starting with the extraction of raw data from the output of diverse molecular simulation packages, moving on to data preprocessing tasks such as decorrelation of time series, using various estimators to derive free energy estimates from simulation samples, and finally providing quality analysis tools for data convergence checking and visualization.
+*alchemlyb* also contains high-level end-to-end workflows that combine multiple building blocks into a user-friendly analysis pipeline from the initial data input stage to the final results. This workflow functionality enhances accessibility by enabling researchers from diverse scientific backgrounds, and not solely computational chemistry specialists, to use *alchemlyb* effectively.
+
+
+# Statement of need
+
+In the pharmaceutical sector, computational chemistry techniques are integral for evaluating potential drug compounds based on their protein binding affinity [@deng2009computations].
+Notably, absolute binding free energy calculations between proteins and ligands or relative binding affinity of ligands to the same protein are routinely employed for this purpose [@merz2010drug].
+The resultant estimates of these free energies are essential for understanding binding affinity throughout various stages of drug discovery, such as hit identification and lead optimization [@merz2010drug].
+Other free energies extracted from simulations are useful in solution thermodynamics, chemical engineering, environmental science, and material science [@Schlaich2015aa].
+
+Molecular simulation packages such as [GROMACS](https://www.gromacs.org/) [@Abraham2015aa], [Amber](https://ambermd.org/) [@Case2005uq], [NAMD](https://www.ks.uiuc.edu/Research/namd/) [@phillips2020scalable], [LAMPPS](https://lammps.org/) [@Thompson2022aa], and [GOMC](https://gomc-wsu.org/) [@Nejahi2021aa] are used to run free energy simulations and many of these packages also contain tools for the subsequent processing of simulation data into free energies.
```

Comment:
I put it here because as @schlaicha suggested because it really is a major code ... and we are hoping to support it soon-ish. We could other codes here if people feel strongly about anything else in particular.

Reply:
Thanks @jhenin, I fixed it in your name.